packages/katello/rubygem-spidr/rubygem-spidr.spec

```diff
@@ -0,0 +1,85 @@
+# template: foreman_plugin
```

Comment:
This isn't a Foreman plugin, but just a regular gem. Please use the `gem` template.

Reply:
![Screenshot from 2024-06-20 16-28-29](https://github.com/theforeman/foreman-packaging/assets/21146741/16b59c09-4908-4dc5-a5f0-385db2d130a8)
These are the templates that I see as available. I was trying to use Default but that doesn't seem to be accepted anymore..

Reply:
`default` should do